package.json

```diff
@@ -74,8 +74,8 @@
     "ngx-base": "^2.3.2",
     "ngx-bootstrap": "^2.0.5",
     "ngx-modal": "^0.0.29",
-    "patternfly": "^3.30.1",
-    "patternfly-ng": "^3.10.1"
+    "patternfly": "3.30.1",
+    "patternfly-ng": "3.10.1"
```

Comment:
@christianvogt changed that because there integration problems

Reply:
@edewit i am not changing the version. i am making it sure that only 3.10.1 version is used here

Reply:
As this project is a library, by not using the caret you will force packaging the 3.10.1 version even though another api compatible version of patternfly may be available already within fabric8ui.

Imagine if we had 10 libs doing this and all requested different, yet API compatible, versions. We would bloat the application.

Of course this assumes patternfly adheres to semantic versioning. And if it didn't we need to address that.

Reply:
@christianvogt agree with you and this change would be reverted, it was done just as an attempt to address https://github.com/openshiftio/openshift.io/issues/3833#issuecomment-399058919 , which din't helped